_rules/auto-play-audio-does-not-exceed-3-seconds-aaa1bf.md

```diff
@@ -0,0 +1,114 @@
+---
+id: aaa1bf
+name: auto-play audio does not exceed 3 seconds
+rule_type: atomic
+description: |
+  auto-play audio content does not output audio for more than 3 seconds
+accessibility_requirements:
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+  - Audio output
+  - Visual output
+authors:
+  - Anne Thyme Nørregaard
+  - Bryn Anderson
+---
+
+## Applicability
+
+This rule applies to any HTML [`audio`](https://html.spec.whatwg.org/multipage/media.html#the-audio-element), [`video`](https://html.spec.whatwg.org/multipage/media.html#the-video-element) or [`source`](https://html.spec.whatwg.org/multipage/embedded-content.html#the-source-element) elements, with a [`src`](https://html.spec.whatwg.org/multipage/media.html#attr-media-src) attribute referencing content with a duration of more than 3 seconds, that has an [`autoplay`](https://html.spec.whatwg.org/multipage/media.html#attr-media-autoplay) attribute equal to `true`, and that has both [`paused`](https://html.spec.whatwg.org/multipage/media.html#dom-media-paused) and [`muted`](https://html.spec.whatwg.org/multipage/media.html#attr-media-muted) attributes equal to `false`.
```

Comment:
The way this reads to me, it seems like you're looking for autoplay on the `src` attribute, and also allowing paused / muted to be on the source element, rather than on the audio or video element. I'd also restrict this, not just to any source element but specifically to one that's a child of an audio or video element.

This applies to all 3 of these rules.